api/v1alpha1/alluxioruntime_types.go

```diff
@@ -213,6 +213,9 @@ type AlluxioRuntimeSpec struct {
 
 	// Manage the user to run Alluxio Runtime
 	RunAs *User `json:"runAs,omitempty"`
+
+	// Manage monitoring for Alluxio Runtime
+	Monitor string `json:"monitor,omitempty"`
```

Comment:
what about the key is `monitoring`, and the type is bool?

Reply:
Monitoring is ok for me,We can do this by specifying different types of monitoring, like prometheus or others.
It will be more common.

Reply:
fixed